pre-docs/bolt_configure_puppetdb.md

```diff
@@ -0,0 +1,79 @@
+
+# Configuring PuppetDB
+
+Configure bolt to connect to PuppetDB
+
+## Why
+
+- [Generate an Inventory
+File](inventory_file_generating.md),
+- [Discover Facts for Targets](writing_plans.md#collect-facts-from-puppetdb)
+- [Make Queries in Plans](writing_plans.md#puppetdb_query)
+
+## PuppetDB Authorization
+
+Bolt can authenticate with PuppetDB through an ssl client certificate or when
```

Comment:
This line makes me wish english had more syntactic sugar for logic statements. I'm not sure how to phrase it better though :P

Reply:
perhaps drop the when used with pe bit.